docs/content/latest/deploy/kubernetes/multi-cluster/gke/helm-chart.md

```diff
@@ -530,4 +533,6 @@ kubectl scale statefulset yb-master --replicas=0 -n yb-demo-us-central1-b \
  --context gke_yugabyte_us-central1-b_yugabytedb2
```

Comment:
To test the resilience of the DB cluster when subjected to the complete failure of one region, set the replica count of the YugabyteDB StatefulSets to 0 for the `us-central1` region.